reporthandling/results/v1/resourcesresults/controls.go

```diff
@@ -28,13 +28,38 @@ func (control *ResourceAssociatedControl) SetName(name string) {
 
 // =============================== Status ====================================
 
+// isOldControl checks if a control has no status set - this is for backward compatibility with old reports
+func (control *ResourceAssociatedControl) isOldControl() bool {
+	return control.Status.InnerStatus == "" && control.Status.InnerInfo == ""
```

Comment:
What if the status is failed? It doesn't have an inner nor substatus

Reply:
https://github.com/kubescape/opa-utils/pull/85/files#diff-a8a59c61445c3bb1a5e65359255d3b0c0c860137c50235cd9210a02f26a7002dR68

an example for "failed"

Reply:
And in the new reports we add info for failed controls?